drivers/cummins_ps0600.json

```diff
@@ -12,9 +12,12 @@
     "S_L1": {"description": "Generator L1 VA", "register": 39, "words": 1, "datatype": "uint16", "multiplier": 1000, "unit": "VA", "typecast": "float"},
     "S_L2": {"description": "Generator L2 VA", "register": 40, "words": 1, "datatype": "uint16", "multiplier": 1000, "unit": "VA", "typecast": "float"},
     "S_L3": {"description": "Generator L3 VA", "register": 41, "words": 1, "datatype": "uint16", "multiplier": 1000, "unit": "VA", "typecast": "float"},
-    "V_L1": {"description": "Generator L1-N voltage", "register": 17, "words": 1, "datatype": "uint16", "multiplier": 1, "unit": "V", "typecast": "float"},
     "V_L2": {"description": "Generator L2-N voltage", "register": 18, "words": 1, "datatype": "uint16", "multiplier": 1, "unit": "V", "typecast": "float"},
     "V_L3": {"description": "Generator L3-N voltage", "register": 19, "words": 1, "datatype": "uint16", "multiplier": 1, "unit": "V", "typecast": "float"},
+    "V_L1": {"description": "Generator L1-N voltage", "register": 17, "words": 1, "datatype": "uint16", "multiplier": 1, "unit": "V", "typecast": "float"},
```

Comment:
This looks good, but not sure why this line moved? (would be good to keep to `V_L1`, `V_L2`, `V_L3` order)

Reply:
Yeah it was a weird move. Probably happened while copy-pasting, good catch ! 
I corrected it in last commit